src/telemetry/metrics/labels.rs

```diff
@@ -254,43 +251,12 @@ impl FmtLabels for DstLabels {
 
 // ===== impl TlsStatus =====
 
-impl FmtLabels for TlsStatus {
+impl FmtLabels for ctx::transport::TlsStatus {
     fn fmt_labels(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self.0 {
+        match *self {
```

Comment:
I'm not sure if the dereference here is strictly necessary after Rust 1.26's [match ergonomics](https://github.com/rust-lang/rfcs/blob/master/text/2005-match-ergonomics.md) changes?